Makefile

```
@@ -180,137 +139,73 @@ build-no-ipc:
+check-typos:
+	typos
 
-## Build all man pages
-build-all-man: docs/stratisd.8 docs/stratis-dumpmetadata.8
+## Remove installed items
+clean: clean-cfg clean-ancillary clean-primary
 
-## Build all stratisd binaries and configuration necessary for install
-build-all: build-all-rust build-all-man
+## Remove installed non-primary tools generated by the build process
+clean-ancillary:
```

checkmake (0.3.2)

[warning] Target body for "clean-ancillary" exceeds allowed length of 5 lines (8).

(maxbodylength)